index.html 

```diff
@@ -0,0 +1,42 @@
+<! doctype html>
+<head>
+	<title> Bubble Tea site</title>
+</head>
+<body>
+	< div class=“topnav”>
+		<img src =“ ” ALT=“logo” Width=600 Height = 400”/>
+		<a class= “active” href=#home”>Home</a>
+		<a href=#news”>News</a>
+		< a href=“about”>About</a>
+		<a href=“contact”>Contact</a>
+	</div>
+	<div>
+	<img src =“”  Alt=“description of image”/>
```

Comment:
space tag issues
Alt needs to be alt
<img> is not self-closing.... so drop the extra "/" at the end